src/js/dp/feedback-form.js

```diff
@@ -0,0 +1,122 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const pageURL = window.location.href;
+    const feedbackURL = "/feedback";
+
+
+    const feedbackFormContainer = document.querySelector(
+        "#feedback-form-page-container"
+    );
+    if (feedbackFormContainer) {
+        feedbackFormContainer.addEventListener("submit", function (e) {
+            e.preventDefault();
+            const fieldErrors = document.querySelectorAll(
+                "#feedback-form-page-container .feedback-form-control__error"
+            );
+            fieldErrors.forEach((fieldError) => {
+                fieldError.classList.remove("feedback-form-control__error");
+            });
+
+            const formErrors = document.querySelectorAll(
+                "#feedback-form-page-container .feedback-form-error"
+            );
+            formErrors.forEach((formError) => {
+                formError.remove();
+            });
+
+            const emailField = document.querySelector("#email-field");
+            const descriptionField = document.querySelector("#description-field");
+            let hasErrors = false;
+
+            if (descriptionField && descriptionField.value === "") {
+                const descriptionError =
+                    '<span class="feedback-form-error" role="alert">Write some feedback</span>';
+                if (!document.querySelector("#description-field-label .feedback-form-error")) {
+                    const descriptionFieldLabel = document.querySelector(
+                        "#description-field-label"
+                    );
+                    descriptionFieldLabel.insertAdjacentHTML(
+                        "afterend",
+                        descriptionError
+                    );
+                }
+                descriptionField.classList.add("feedback-form-control__error");
+                descriptionField.focus();
+                hasErrors = true;
+            }
+
+            if (emailField && emailField.value !== "") {
+                let emailError = "";
+                if (hasErrors) {
+                    emailError =
+                        '<span class="feedback-form-error" role="alert" aria-live="polite">This is not a valid email address, correct it or delete it</span>';
```

Comment:
Think we should create the element natively using the [createElement api ](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement). Then the only change would be in the logic to have `aria-live="polite"` or not